Window CI is broken

https://github.com/ddnet/ddnet/pull/4657#issuecomment-1030106374

```
  mkdir debug
  cd debug
  cmake --version
  cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Debug -Werror=dev -DDOWNLOAD_GTEST=ON -DDEV=ON -DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG=. ..
  cmake --build . --config Debug --target everything 
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    CFLAGS: /WX
    CXXFLAGS: /WX
    LDFLAGS: /WX

    Directory: D:\a\ddnet\ddnet

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----            2/4/2022  3:43 PM                debug
cmake version 3.22.2

CMake suite maintained and supported by Kitware (kitware.com/cmake).
CMake Error at CMakeLists.txt:42 (project):
  Generator

    Visual Studio 16 2019

  could not find any instance of Visual Studio.



-- Configuring incomplete, errors occurred!
See also "D:/a/ddnet/ddnet/debug/CMakeFiles/CMakeOutput.log".
CMake Error:
  Generator

    Visual Studio 16 2019

  could not find any instance of Visual Studio.

```

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -42,7 +42,7 @@ jobs:
             CFLAGS: -Wdeclaration-after-statement -Werror
             CXXFLAGS: -Werror
         - os: windows-latest
-          cmake-args: -G "Visual Studio 16 2019" -A x64
+          cmake-args: -G "Visual Studio 17 2022" -A x64
           package-file: DDNet-*-win64.zip
           fancy: false
           env:

